ocaml/xcp-rrdd/bin/rrdd/xcp_rrdd.ml

```diff
@@ -775,6 +775,11 @@ let monitor_write_loop writers =
               with_lock Rrdd_shared.last_loop_end_time_m (fun _ ->
                   Rrdd_shared.last_loop_end_time := Unix.gettimeofday ()
               ) ;
+
+              (* Dump to /dev/shm/metrics/host-dss *)
+              let path = Rrdd_server.Plugin.get_path "host-dss" in
+              Rrdd_server.dump_host_dss_to_file path ;
```

Comment:
I would prefer if this was done inside the do_monitor_write timed section, after the update_rrds is executed

Reply:
OK will move there.

Reply:
Fixed now.